server/graphql/v2/object/Host.js

```diff
@@ -64,6 +67,46 @@ const getNumberOfDays = (startDate, endDate, host) => {
   return days(since, endDate || undefined);
 };
 
+const convertCurrencyAmount = async (fromCurrency, toCurrency, date, amount) => {
+  const fxRate = await getFxRate(fromCurrency, toCurrency, date);
+  const convertedAmount = Math.round(amount * fxRate);
+  return { date: date, amount: convertedAmount };
+};
+
+const amountsOverTimeInHostCurrency = async (
+  host,
+  transactionKind,
+  transactionType,
+  timeUnit,
+  collectiveIds,
+  dateFrom,
+  dateTo,
+) => {
+  const transactionDatePoints = await queries.getTransactionsTimeSeries(
+    transactionKind,
+    transactionType,
+    host.id,
+    timeUnit,
+    collectiveIds,
+    dateFrom,
+    dateTo,
+  );
+  const dataPointsInHostCurrency = transactionDatePoints.map(async contributionAmount => {
+    const contributionAmountInHostCurrency = await convertCurrencyAmount(
+      contributionAmount.currency,
+      host.currency,
+      contributionAmount.date,
+      contributionAmount.amount,
+    );
```

Comment:
Maybe we should be confident that  the hostCurrency in the ledger is the host.currency? And no need to convert? Could always throw an Error if that's not the case.

Reply:
I've simplified the logic to remove this conversion as I think it's not longer needed as we are now getting the `amountInHostCurrency`. This part I only added when we were using the previous query which had `amount` instead of `amountInHostCurrency`.